common/tools/perf-tests-track-1-setup.js

```diff
@@ -0,0 +1,19 @@
+const { spawnSync } = require("child_process");
+
+function spawn(command) {
+  const ret = spawnSync(command, { shell: true, stdio: "inherit" });
+
+  if (ret.status !== 0) {
+    throw new Error(`${command} failed with exit code ${ret.status}`);
+  }
+}
+
+const navigateToPerfStressFolder = `cd ../../../test-utils/perfstress`;
```

Comment:
this feels slightly risky for cross-plat, but it works on windows/linux/mac right?

Reply:
Hmm... Worked fine on windows, should definitely work in linux/mac.
But I can move to "path.join" to be safe.